automatix/bundlewrap.py

```diff
@@ -27,3 +29,12 @@ def _get_remote_hostname(self):
             return system.replace('hostname!', '')
         node = self.env.config['bw_repo'].get_node(system)
         return node.hostname
+
+
+class BWNodesWrapper:
+    def __init__(self, repo: Repository, systems: dict):
+        self.repo = repo
+        self.systems = systems
+
+    def __getattr__(self, name):
```

Comment:
Just recognized, that this seems to be a Python3.7 feature: We should update the requirements here!

Reply:
FTR: Python3.7 feature is for modules. So it also works this way in Python3.6. No need for requirement adjustment. I am going to revert the commit.